### PR TITLE
Fix/traefik logrotate error

### PR DESCRIPTION
**Pull request by sehqlr**
This fixes a problem that prevented the rpm from being properly installed. Also, I fixed an error in one of the logrotate config files

### CODE REVIEW

File: packages/traefik/spec.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 name: traefik
 version: 0.0.0
-epoch: 341
-iteration: 7
+epoch: 412
+iteration: 9
```
Comment (BrianHicks) [inline packages/traefik/spec.yml]: should this be 8?

Comment (stevendborrelli) [inline packages/traefik/spec.yml]: It was bumped to 8 in https://github.com/asteris-llc/mantl-packaging/commit/ef46746ffbccc81dea2bd25997f7e429b242791e (which was never merged to master).

Comment (sehqlr) [inline packages/traefik/spec.yml]: I think this was from before 7 and 8 were merged to 7

Comment (BrianHicks) [inline packages/traefik/spec.yml]: Well in that case, we should only go up one. 8 will work fine.

**Comment by stevendborrelli** [comment]
I've changed the iteration to 8

**Comment by BrianHicks** [comment]
LGTM :thumbsup:

**Comment by BrianHicks** [comment]
also, why does this disable the Consul build?

**Comment by sehqlr** [comment]
My fault, that commit needs to be reverted

**Pull request merged**